.github/workflows/build.yml

```
@@ -80,11 +80,26 @@ jobs:
       - uses: hashicorp/setup-terraform@v2
         with:
           terraform_version: ${{ steps.setup-env.outputs.terraform-version }}
+      - name: Install go-critic
+        env:
+          PACKAGE_URL: github.com/go-critic/go-critic/cmd/gocritic
+          PACKAGE_VERSION: ${{ steps.setup-env.outputs.go-critic-version }}
+        run: go install ${PACKAGE_URL}@${PACKAGE_VERSION}
+      - name: Install GoSec
```

Aligns with the project's capitalization and the capitalization used in [cisagov/setup-env-github-action](https://github.com/cisagov/setup-env-github-action).
```suggestion
      - name: Install gosec
```